drivers/azure/azureutil/azureutil.go

```diff
@@ -159,6 +160,9 @@ func (a AzureClient) CreatePublicIPAddress(ctx *DeploymentContext, resourceGroup
 			Location: to.StringPtr(location),
 			Properties: &network.PublicIPAddressPropertiesFormat{
 				PublicIPAllocationMethod: ipType,
+				DNSSettings: &network.PublicIPAddressDNSSettings{
+					DomainNameLabel: to.StringPtr(name),
```

Comment:
why do you need to add a FQDN? if the ip-name is taken "globally" this will fail. For instance if two of us ask for a machine named foo in westus, one of us will get a failure because FQDN is used. That'll just surface more errors to the users, unless you need a FQDN, let's avoid it.